docs/Formula-Cookbook.md

```diff
@@ -1052,6 +1052,24 @@ The `sockets` method accepts a formatted socket definition as `<type>://<host>:<
 
 Please note that sockets will be accessible on IPv4 and IPv6 addresses by default.
 
+If you only need one socket and you don't care about the name (the default is `listeners`):
+
+```rb
+service do
+  run [opt_bin/"beanstalkd", "test"]
+  sockets "tcp://127.0.0.1:80"
+end
+```
+
+If you need multiple sockets and/or you want to specify the name:
+
+```rb
+service do
+  run [opt_bin/"beanstalkd", "test"]
+  sockets socket: "tcp://0.0.0.0:80", socket_tls: "tcp://0.0.0.0:443"
```

Comment:
```suggestion
  sockets http: "tcp://0.0.0.0:80", https: "tcp://0.0.0.0:443"
```
or something perhaps so these names are a bit more meaningful?

alternatively, if we want to just generate the names for people:
```suggestion
  sockets ["tcp://0.0.0.0:80", "tcp://0.0.0.0:443"]
```

Reply:
I'll update the names in the docs to be more meaningful like you mentioned. I think allowing names can be helpful for documentation reasons but I don't feel very strongly either way.